tests/components/zha/test_websocket_api.py

```diff
@@ -318,8 +317,8 @@ async def test_update_zha_config(
 
     await zha_client.send_json({ID: 6, TYPE: "zha/configuration"})
     msg = await zha_client.receive_json()
-    configuration = msg["result"]
-    assert configuration == configuration
+    test_configuration = msg["result"]
+    assert test_configuration == configuration
```

Comment:
Hello @dmulcahey, @adminiuga, @puddly, @TheJulianJES, can you look at this test? There is a bug where the result overwrote the configuration sent und then compared it with itself, which of course always passed. But when fixed the test now fails. Thanks in advance!

Reply:
https://github.com/home-assistant/core/pull/114495 should address this.

Reply:
Thank you!